pkgs/top-level/aliases.nix

```diff
@@ -840,6 +842,7 @@ mapAliases {
   nextcloud27Packages = throw "Nextcloud27 is EOL!"; # Added 2024-06-25
   nagiosPluginsOfficial = monitoring-plugins;
   neochat = libsForQt5.kdeGear.neochat; # added 2022-05-10
+  nerdfonts = throw "nerdfonts has been separated into individual font packages under the namespace nerd-fonts"; # Added 2024-11-09
```

Comment:
> What exactly should that PR add/change?

This `throw` line.

Reply:
Sure I'll do that later

Reply:
Just additional notes: this can hardly be made really backward-compatible without more cumbersome workarounds, because the font files are now inside the `<fontDirName>` sub-directory of `$out/share/fonts/{opentype,truetype}/NerdFonts`, rather than directly inside the latter. Despite of this, simple "end use" with automatic indexing should not fail.

Reply:
I think that what @lunagl wanted is simply that the `nerdfonts` attribute will be evaluated to a `symlinkJoin` derivation that incldues all fonts... I think we discussed this here, and we decided not to do that eventually.